article_store/articles/models.py

```diff
@@ -0,0 +1,84 @@
+import re
+import uuid
+
+from django.core.validators import RegexValidator
+from django.db import models
+
+
+PREVIEW = 'preview'
+PUBLISHED = 'published'
+READY = 'ready'
+
+ARTICLE_STATUSES = (
+    (PREVIEW, 'preview'),
+    (PUBLISHED, 'published'),
+    (READY, 'ready'),
```

Comment:
not sure what is `ready` (the other two are clear)?

Reply:
Statuses in general can be ignored for now, given the API creates an immediately published article version